### PR TITLE
Fix to allow admins to edit post payment

### PR DESCRIPTION
**Pull request by beingmattlevy**
Fixes logic to allow admins to edit after payment
Fixes display logic for admins to see who request is and who they are signed in as

### CODE REVIEW

File: app/views/ticket_requests/_form.html.haml [hunk 1]
```diff
@@ -84,7 +85,7 @@
               -# on create, set user into guests list
               = f.hidden_field :guests, value: @user.name_and_email
 
-            - unless @ticket_request.post_payment?
```
Comment (kigster) [inline app/views/ticket_requests/_form.html.haml:87]: I think you can do here:

```ruby
unless post_payment? || event.admin?(current_user)
```

**Review by kigster — CHANGES_REQUESTED** [review]
It might be easier to just pull up the admin comments to the top of the form?

**Comment by beingmattlevy** [comment]
It is not just the admin notes that potentially needs editing. Admins may have to adjust other fields.

File: app/views/ticket_requests/_form.html.haml (hunk 1 above)
Comment (beingmattlevy) [reply app/views/ticket_requests/_form.html.haml:87]: This does not work. The unless gets caught in the first eval of post_payment and then stops.

**Review by kigster — APPROVED** [review]

**Pull request merged**